lib/facter/grafana_version.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Facter.add(:grafana_version) do
+  confine { Facter::Util::Resolution.which('grafana-server') }
```

Comment:
shouldn't we confine on `/usr/share/grafana/VERSION`?

Reply:
ja, yes, maybe, wanted to make sure that the binary is also there, so that maybe the version isn't only a leftover or so... but yes, can confine on the file 🤷